setuptools/tests/test_namespaces.py

```diff
@@ -109,3 +109,34 @@ def test_namespace_package_installed_and_cwd(self, tmpdir):
         ]
         with test.test.paths_on_pythonpath([str(target)]):
             subprocess.check_call(pkg_resources_imp, cwd=str(pkg_A))
+
+    @pytest.mark.skipif(bool(os.environ.get("APPVEYOR")),
```

Comment:
No need for the `bool` here.

Also, can you make this `xfail`? This test *should* pass on Appveyor, so it would be good to know if it changes from `xfail` to `xpass`.

Reply:
I noticed that in appveyor run for this pull request 3 of 4 namespace tests passed when running under python-2.7.15 and all 4 passed when running under python-3.6.6. Should I remove ``xfail`` for them?

Reply:
@daa The rest of the repo is not super consistent about `xfail` vs `skip`. Going forward we're trying to make a clear distinction, and at some point I'll go through and try and clean it up so that we can turn on strict `xfail` (e.g. if an `xfail` test succeeds, that's a failure).

Reply:
Oh sorry, I misunderstood. Yeah, I think you should change these so that you don't introduce any new `XPASS`es.

Reply:
Ok, I removed ``xfail`` marks for namespace tests that pass on AppVeyor.